.eslintrc

```diff
@@ -1,21 +1,18 @@
 {
-    "parserOptions": {
-        "ecmaVersion": 2019,
-        "sourceType": "module",
-        "ecmaFeatures": {
-            "jsx": true
-        }
-    },
-    "extends": [
-        "eslint:recommended",
-        "plugin:react/recommended",
-        "prettier"
-    ],
-    "rules": {
-        "strict": ["error", "never"]
-    },
-    "env": {
-        "node": true,
-        "browser": true
+  "parserOptions": {
+    "ecmaVersion": 2019,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
     }
-}
\ No newline at end of file
+  },
+  "extends": ["plugin:react/recommended", "prettier"],
```

Comment:
After testing this locally with the original rules `"eslint:recommended", "plugin:react/recommended", "prettier"`, I believe we should keep this as it originally was. Meanwhile, the `no-debugger` rule makes sense to move forward on as it's either on or off. There is no `warning` option for this feature that I could locate.